### ISSUE TITLE
Add debouncing to changes on inputs

### ISSUE THREAD

**Issue by M-J-Robbins** [email-markup-consortium/email-reader-view#7]
`Uncaught (in promise) Error: This request exceeds the MAX_WRITE_OPERATIONS_PER_MINUTE quota.`

Too many changes cause a crash, this is coming from `addEventListener('change',` [in the code here](https://github.com/email-markup-consortium/email-reader-view/blob/main/popup.js#L340-L343)

If we swap `change` to `input` it stops the error. However having this in real time is a great feature. So we can maybe look at [throttling or debouncing](https://www.telerik.com/blogs/debouncing-and-throttling-in-javascript) it somehow. 

As far as my understanding goes, I think debouncing makes more sense.

**Comment by theotherstevenc** [comment]
What steps will replicate the crash you're experiencing?

Since [L340-L343](https://github.com/email-markup-consortium/email-reader-view/blob/main/popup.js#L340-L343) is only responsible for updating the UI to reflect the chosen values, and doesn't affect the the rendered email, it _could_ simply be removed.

**Comment by M-J-Robbins** [comment]
Yeah sorry a bit short on details, I wrote it quickly last night. 

To reproduce
* Open an email
* turn on reader view
* change the background colour, drag the picker across a range of colours.
* The UI will still update as expected but at some point the live updates in the email will freeze
* An error will show next to the reader view on the extensions page chrome://extensions/

Tested in Chrome 108, MacOS 12.6.1. On Gmail, Yahoo, Outlook
